src/shapes/textbox.class.ts

```diff
@@ -293,6 +304,83 @@ export class Textbox extends IText {
     return value.split(this._wordJoiners);
   }
 
+  /**
+   * Wraps a line of text using the width of the Textbox and a context. For overflowBreakWord option
+   * @param {Array} line The grapheme array that represent the line
+   * @param {Number} lineIndex
+   * @param {Number} desiredWidth width you want to wrap the line to
+   * @param {Number} reservedSpace space to remove from wrapping for custom functionalities
+   * @returns {Array} Array of line(s) into which the given text is wrapped
+   * to.
+   */
+  _wrapLineOfWordBreak(
```

Comment:
I think it is best to add in the comment an explanation of how the method works
Something like:
This method splits text into lines by looking for the last space in a measured piece of text that fits the bounds. 
If a space is found, the line breaks before the last word in the sequence.
If not, it splits the last word and breaks the line.

Reply:
Very helpful !